tests/dummy/app/controllers/ember-flexberry-dummy-application-user-edit.js

```diff
@@ -1,6 +1,7 @@
-import BaseEditFormController from './base-edit-form';
+import BaseEditFormController from 'ember-flexberry/controllers/edit-form';
+import EditFormMixin from '../mixins/base-edit-form-mixin';
```

Comment:
Please rename mixin file to 'edit-form-controller-handle-operations'.
And inport it then as 'EditFormControllerHandleOperationsMixin' here and in all other edit-forms in dummy application.